opentelemetry-semantic-conventions/scripts/templates/registry/rust/metric.rs.j2

```diff
@@ -0,0 +1,79 @@
+{%- import 'macros.j2' as metric_macros -%}
+// DO NOT EDIT, this is an auto-generated file
+//
+// If you want to update the file:
+// - Edit the template at scripts{{template}}
```

Comment:
some issue with auto-generation, as file name is missing?

Reply:
Fixed in [4970ee9](https://github.com/open-telemetry/opentelemetry-rust/pull/2098/commits/4970ee9314ca5cb72376b21bb807ec14da0d4cc8)